components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -519,29 +520,32 @@ void KeyringController::AddImportedAccount(
     std::move(callback).Run(false, "");
     return;
   }
-  if (!default_keyring_) {
-    std::move(callback).Run(false, "");
-    return;
-  }
 
-  const std::string address = default_keyring_->AddImportedAccount(private_key);
-  if (address.empty()) {
+  const std::string address =
+      AddImportedAccountForDefaultKeyring(account_name, private_key);
+
+  std::move(callback).Run(!address.empty(), address);
+}
+
+void KeyringController::AddImportedAccountByJson(
```

Comment:
`AddImportedAccountFromJson`?

Reply:
I guess really `ImportAccountFromJSON`?

Reply:
fixed in 8ef138a5650682db54f7b63468334df477ed4d37